helpers/helpers_emba_prepare.sh

```diff
@@ -458,10 +459,10 @@ detect_root_dir_helper() {
     R_PATH=$(echo "$R_PATH" | awk '{print $2}')
     if [[ -d "$R_PATH" ]]; then
       ROOT_PATH+=( "$R_PATH" )
-      if [[ -n "$MECHANISM" ]] && ! echo "$MECHANISM" | grep -q "dir names"; then
-        MECHANISM="$MECHANISM / dir names"
-      elif ! echo "$MECHANISM" | grep -q "binary interpreter"; then
+       if [[ -z "$MECHANISM" ]]; then
```

Comment:
alignement issue? Which editor are you using? Please set it up to automatically use 2 spaces for a tab